bugbug/tools/code_review.py

```diff
@@ -56,25 +56,37 @@ class HunkNotInPatchError(ModelResultError):
     """Occurs when the hunk in the model result is not part of the patch."""
 
 
-PROMPT_TEMPLATE_SUMMARIZATION = """You are an expert reviewer for the Mozilla Firefox source code, with experience on source code reviews.
+COND_MENTION_MOZILLA = False
```

Comment:
```suggestion
TARGET_SOFTWARE= None
```
and then replace `(" the Mozilla Firefox" if COND_MENTION_MOZILLA else "")` with `(f" the {TARGET_SOFTWARE}" if TARGET_SOFTWARE is not None else "")`

This way it's more generic.